<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateItem</class>
 <widget class="QDialog" name="CreateItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Criar Item/Produto | Sistema de Controle de Comandas</string>
  </property>
  <widget class="QGroupBox" name="groupBox_itemCreate">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>110</y>
     <width>491</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_itemCreate">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>327</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Criar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_itemCreateCancel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>327</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>22</y>
      <width>461</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_itemCreate">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_itemName">
       <item>
        <widget class="QLabel" name="label_itemName">
         <property name="text">
          <string>Nome</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_itemName">
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_itemDescription">
       <item>
        <widget class="QLabel" name="label_itemDescription">
         <property name="text">
          <string>Descrição</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_itemDescription"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_itemAmount">
       <item>
        <widget class="QLabel" name="label_itemAmount">
         <property name="text">
          <string>Quantidade</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_itemAmount">
         <property name="inputMask">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_itemValue">
       <item>
        <widget class="QLabel" name="label_itemValue">
         <property name="text">
          <string>Valor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_itemValue">
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="maxLength">
          <number>6</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_itemCreateTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Criar Item/Produto</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
